metrics/web/docs/features/public_dashboard/01_public_dashboard.md

```diff
@@ -137,3 +158,187 @@ Once we've chosen the implementation approach, let's consider the components dia
+
+Let's review the following class diagram, which describes the relationship between the classes in this module:
+
+![Feature Config Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/Flank/flank-dashboard/public_dashboard_design/metrics/web/docs/features/public_dashboard/diagrams/public_dashboard_feature_config_class_diagram.puml)
+
+#### Auth module
+
+The next one, we should add the ability to sign-in anonymously. Also, we should know whether the user sign-in anonymously. For these purposes, we should make the following changes in the `auth` module:
+- add the `signInAnonymously()` method to the `UserRepository` abstract class;
+- add the implementation of the `signInAnonymously()` method to the `FirebaseUserRepository` class;
+- create the [`SignInAnonymouslyUseCase`](#SignInAnonymouslyUseCase) class, which calls the `signInAnonymously()` method of the `UserRepository`;
+- create the [`UserProfileViewModel`](#UserProfileViewModel) class;
+- create the `AuthState` enum;
+- integrate the use case, the enum, and the view model described above into the `AuthNotifier`.
+
+Let's review the following class diagram, which describes the relationship between the classes in this module:
+
+![Auth Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/Flank/flank-dashboard/public_dashboard_design/metrics/web/docs/features/public_dashboard/diagrams/public_dashboard_auth_class_diagram.puml)
+
+#### Navigation module
+
+The last thing, we should have the ability to manage the navigation of the anonymous user, that is, to know which pages they can visit and which they can't. Following the next steps, helps to achieve this purpose:
+- add the field of the `AuthState` type to the `NavigationNotifier`;
+- change the `handleAuthenticationUpdates()` method of the `NavigationNotifier` to take an argument of the type `AuthState`;
+- create a set of the pages' `RouteName`, which the anonymous user can visit (at the moment, it is only the `dashboard` route name);
+- add the additional condition to the `_processConfiguration()` method, which should state the following, if the user auth state is logged in anonymously, and they visit the page, whose `RouteName` is not in the anonymous `RouteName` set, then redirect to the `login` page;
+
+#### Making things work
+
+After the changes in the modules above, let's move on to integrating these changes into the application.
+
+Here are the main parts that we should add to the application:
+- auto sign in the user anonymously if the feature is enabled;
+- manage normal sign in for the anonymous user;
+- change the UI of the Metrics user menu.
+
+Let's review each of the above points in the following subsections
+
+##### Auto sign in the user anonymously
+
+To auto sign in the user anonymously, we should make the following:
```

Comment:
Consider using a numbered list, according to the [Technical Writing](https://developers.google.com/tech-writing/one/lists-and-tables#choose_the_correct_type_of_list) article suggestions.